config/api.yml

```diff
@@ -739,6 +739,23 @@
         :identifier: storage_tag
       - :name: unassign
         :identifier: storage_tag
+  :event_streams:
+    :description: Event Streams
+    :options:
+    - :collection
+    :verbs: *gp
+    :klass: EventStream
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: foo
```

Comment:
AFAICT there are no product features for event streams/miq events. Should these be access-to-all or do we need to add some?

Reply:
I'd say we want identifier for those, especially if accessing via the main collection endpoint, probably something similar to Metrics at the end of the miq_product_features.yaml.  As subcollection, we could leverage the role identifier for viewing the resources themselves.

/cc @h-kataria thoughts ?